dali/operators/reader/loader/numpy_loader.h

```diff
@@ -36,21 +36,25 @@
 
 namespace dali {
 
-TypeInfo TypeFromNumpyStr(const std::string &format);
+const TypeInfo &TypeFromNumpyStr(const std::string &format);
 
 class NumpyParseTarget{
  public:
   std::vector<int64_t> shape;
-  TypeInfo type_info;
+  const TypeInfo *type_info;
```

Comment:
Shouldn't we initialize it with nullptr?

Reply:
I guess we should - that, and other members.

Reply:
Done.